test/images/port-forward-tester/Makefile

```diff
@@ -0,0 +1,15 @@
+all: push
+
+TAG = 1.0
+
+portforwardtester: portforwardtester.go
+	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-w' ./portforwardtester.go
+
+image: portforwardtester
+	sudo docker build -t ncdc/portforwardtester:$(TAG) .
+
+push: image
+	sudo docker push ncdc/portforwardtester:$(TAG)
```

Comment:
Please remove the `sudo` and make this `gcloud docker push gcr.io/google_containers/portforwardtester:$(TAG)`